packages/loader/src/ShaderLoader.ts

```
@@ -0,0 +1,56 @@
+            params: { shaderVirtualPath }
+          });
+        })
+      ).then(() => {
+        return Shader.create(code);
+      });
+    });
+  }
+
+  private getBuiltinShader(code: string) {
+    const match = code.match(ShaderLoader._builtinRegex);
+    if (match && match[1]) return match[1];
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)